### ISSUE TITLE
The encoding test should be reviewed

### ISSUE THREAD

**Issue by heidivanparys** [inspire-eu-validation/data#35]
There are two issues with the encoding test:

- Encoding should allow gmx:Anchor too (besides gco:CharacterString): https://github.com/inspire-eu-validation/ets-repository/issues/127,  and in general: take substitution groups into account in the XPaths

- The guidance regarding encoding is actually unclear (GML vs. the actual application schema (or both maybe?); is specification required or not): https://github.com/inspire-eu-validation/ets-repository/issues/30

**Comment by cportele** [comment]
I agree. I think that resolving the two referenced issues should also resolve this issue. The 2016.3 sub-group should discuss and decide what should be done.

**Comment by heidivanparys** [comment]
@michellutz I think this issue should have the "for-2017.4-discussion" label.

**Comment by michellutz** [comment]
Implementation of the agreed resolutions for the two issues is progressing. I therefore propose to remove the "for-2017.4-discussion" label.
